syntax = "proto2";

import "gpu/libcecl/proto/libcecl.proto";

package datasets.benchmarks.gpgpu;

// Recording of a single GPGPU benchmark execution.
message GpgpuBenchmarkRun {
  required string benchmark_suite = 1;
  required string benchmark_name = 2;
  required string dataset_name = 3;
  required string hostname = 4;
  required gpu.libcecl.LibceclExecutableRun run = 5;
}

